lib/synapse/version.rb

```diff
@@ -1,3 +1,3 @@
 module Synapse
-  VERSION = "0.16.1"
+  VERSION = "0.16.2"
```

Comment:
can we bump the gem version in a separate, last commit in this PR? so we can see in logs later what the version contains.